documentation/projects.adoc

```diff
@@ -20,11 +17,11 @@ import=eclipse,intellij
 active=true
 ----
 
- .Variables of project import
-[options="header"]
-
-|=== |*Variable*|*Value*|*Meaning*
-|`path`|e.g. `myproject`, will clone into `${WORKSPACE_PATH}/myproject`|(required) Path into which the projects is cloned.
+.Variables of project import
+[options="header",cols="3"]
+|===
```

Comment:
This is the main spot that caused the error in `projects.adoc` and `License.adoc`